npm/vite-dev-server-fresh/src/getVite.ts

```diff
@@ -0,0 +1,23 @@
+import debugFn from 'debug'
+import type { ViteDevServerConfig } from './devServer'
+
+const debug = debugFn('cypress:vite-dev-server:getVite')
+
+let vite: typeof import('vite')
```

Comment:
I think this binding might technically work, but having this side-effect thing in the module feels super weird, can we just have `initVite` be named `getVite`

```ts
return require(viteImportPath)
```

and just pass the result `vite` module returned into the functions that use it like we do w/ webpack

Reply:
Yeah I can do that, thought the side-effect was cheeky but I'll update it, would be nice to have consistency between the dev-servers

Reply:
Yeah I probably shoulda said "clever" instead of "super weird" haha, we've already overspent the "clever/weird" budget in the `require.resolve` stuff, gotta keep things simple somewhere 😛

Reply:
Updated in [bee4eb4](https://github.com/cypress-io/cypress/pull/21091/commits/bee4eb4609237702f2781025ce5c3e7e313644b1)